src/app/content/search/utils.ts

```diff
@@ -222,7 +236,9 @@ export const getFilteredResults = (searchResults: SearchResult) => ({
   ...searchResults,
   hits: {
     ...searchResults.hits,
-    hits: searchResults.hits.hits.filter((hit) => !matchKeyTermHit(hit) || !!hit.highlight.title)},
+    hits: searchResults.hits.hits.filter((hit) =>
+      matchAcceptedDocTypeHit(hit) && (!matchKeyTermHit(hit) || !!hit.highlight.title)
```

Comment:
In the future, when we're no longer receiving key terms results we don't want, we can just rely on the first condition

Reply:
leave this in a code comment